### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by amashutikov**
[DEMO LINK](https://amashutikov.github.io/react_todo-app-with-api/)

### CODE REVIEW

**Comment by amashutikov** [comment]
ddwq

**Comment by amashutikov** [comment]
https://mate-academy.github.io/react_todo-app-with-api/
It works the same way in working example btw.

File: src/types/Filters.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export enum TodoFilter {
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
```
Comment (dffUqp) [inline src/types/Filters.ts:4]:
```suggestion
  ALL = 'all',
  ACTIVE = 'active',
  COMPLETED= 'completed',
```

File: src/components/TodoEditForm.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    }
+  };
+
+  useEffect(() => {
+    window.addEventListener('keyup', handleEscapePress);
+
+    return () => {
+      window.removeEventListener('keyup', handleEscapePress);
+    };
+  }, []);
```
Comment (dffUqp) [inline src/components/TodoEditForm.tsx:39]: let's handle keyup in our input.

input has `onKeyUp` prop

**Pull request closed without merging**